gobblin-modules/gobblin-kafka-09/src/test/java/org/apache/gobblin/runtime/KafkaAvroJobStatusMonitorTest.java

```diff
@@ -358,6 +370,66 @@ public void testProcessingRetriedForApparentlyTransientErrors() throws IOExcepti
     jobStatusMonitor.shutDown();
   }
 
+  @Test (dependsOnMethods = "testProcessingRetriedForApparentlyTransientErrors")
+  public void testProcessMessageForCancelledAndKilledEvent() throws IOException, ReflectiveOperationException {
```

Comment:
sorry, can't recall: was this in the prior revision or just introduced?

I'm not clear whether it belongs in this PR or is unrelated...

Reply:
I realized that the tests were out of order in their dependsOn after I added more tests so I fixed that in the file, sorry for the confusion

Reply:
no worries, makes sense